Modules/Quotas/src/Quotas.Application/DTOs/TierDetailsDTO.cs

```diff
@@ -8,10 +8,14 @@ public TierDetailsDTO(Tier tier, IEnumerable<Metric> metrics)
     {
         Id = tier.Id;
         Name = tier.Name;
-        Quotas = tier.Quotas.Select(quota => new TierQuotaDefinitionDTO(quota.Id,
-            new MetricDTO(quota.MetricKey, metrics.First(metric => metric.Key == quota.MetricKey).DisplayName),
-            quota.Max, quota.Period));
-
+        Quotas = tier.Quotas.Select(quota =>
+            new TierQuotaDefinitionDTO(
+                quota.Id,
+                new MetricDTO(metrics.First(metric => metric.Key == quota.MetricKey)),
+                quota.Max,
+                quota.Period
+            )
+        );
```

Comment:
There is a convention that says that in lambda expressions you should use single letter variables. And since the scope of variables in lambdas is so small, I'm totally okay with that. I adhere to that convention wherever it makes sense. 
```suggestion
        Quotas = tier.Quotas.Select(q =>
            new TierQuotaDefinitionDTO(
                q.Id,
                new MetricDTO(metrics.First(metric => m.Key == q.MetricKey)),
                q.Max,
                q.Period
            )
        );
```